include/core/surface_tension_model.h

```diff
@@ -0,0 +1,112 @@
+/* ---------------------------------------------------------------------
+ *
+ * Copyright (C) 2019 -  by the Lethe authors
+ *
+ * This file is part of the Lethe library
+ *
+ * The Lethe library is free software; you can use it, redistribute
+ * it, and/or modify it under the terms of the GNU Lesser General
+ * Public License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * The full text of the license can be found in the file LICENSE at
+ * the top level of the Lethe distribution.
+ *
+ * ---------------------------------------------------------------------
+ */
+
+#ifndef lethe_surface_tension_model_h
+#define lethe_surface_tension_model_h
+
+#include <core/physical_property_model.h>
+
+/**
+ * @brief SurfaceTensionModel. Abstract class that allows to calculate the
+ * surface tension coefficient.
+ */
+class SurfaceTensionModel : public PhysicalPropertyModel
```

Comment:
You're right, I'll rework this this afternoon, I'd like to make it a clean implementation rather than something that we'll have to work on again eventually ! :)